can/interfaces/slcan.py

```diff
@@ -203,7 +203,7 @@ def _recv_internal(
 
         if not string:
             pass
-        elif string[0] == "T":
+        elif string[0] == "T" or string[0] == "x":
```

Comment:
```suggestion
        elif string[0] in ("T", "x"):
```
You should also add a comment that explains, where the x comes from.
A test would be great too.